src/Microsoft.FeatureManagement.AspNetCore/TargetingHttpContextMiddleware.cs

```diff
@@ -50,14 +52,15 @@ public async Task InvokeAsync(HttpContext context, ITargetingContextAccessor tar
 
             if (targetingContext != null)
             {
-                context.Items[TargetingIdKey] = targetingContext.UserId;
+                var activityFeature = httpContext.Features.Get<IHttpActivityFeature>();
```

Comment:
Can it be null?

Reply:
Seems like yes, it can be. Adjusting the following line from:
```csharp
activityFeature.Activity.AddBaggage(TargetingIdKey, targetingContext.UserId);
```
To:
```csharp
activityFeature?.Activity.AddBaggage(TargetingIdKey, targetingContext.UserId);
```